spec/index.bs

```diff
@@ -3004,6 +3021,17 @@ Issue: need a more detailed validation of the render states.
 
 Issue: need description of the render states.
 
+### {{GPUDevice/createReadyRenderPipeline()}} ### {#device-createReadyRenderPipeline}
+
+Same as {{GPUDevice/createRenderPipeline()}}, but returns its result as a
+promise which doesn't resolve until the pipeline is ready to be used without
+additional delay.
+
+Issue: Define fully.
```

Comment:
What happens if there is a validation error?

Reply:
Right, good question. I see three options:

1. Have it behave like usual - still resolves to a pipeline object that is internally invalid; raise a validation error asynchronously.
2. Have it reject with the validation error.
3. Have it reject with OperationError but also raise the full validation error asynchronously.

I can see benefits to both. 2 is less surprising, but 1 integrates with the rest of the error handling we have set up (especially telemetry/onuncapturederror). So maybe 3 is best?

Reply:
Talked on the editors meeting, got convinced that (1) is the way to go.

Reply:
On these principles I think:
- Raising the same error in two different ways (3) means the app has to handle both.
- 2 doesn't compose with telemetry/onuncapturederror.
- Pipeline creation failure is a programmer error and shouldn't be happening in production.